packages/gatsby/src/utils/flags.ts

```diff
@@ -130,34 +129,6 @@ const activeFlags: Array<IFlag> = [
       }
     },
   },
-  {
-    name: `LAZY_IMAGES`,
-    env: `GATSBY_EXPERIMENTAL_LAZY_IMAGES`,
-    command: `develop`,
-    telemetryId: `LazyImageProcessing`,
-    experimental: false,
-    description: `Don't process images during development until they're requested from the browser. Speeds starting the develop server. Requires gatsby-plugin-sharp@2.10.0 or above.`,
-    umbrellaIssue: `https://gatsby.dev/lazy-images-feedback`,
-    noCI: true,
-    testFitness: (): fitnessEnum => {
-      // Take a 10% of slice of users.
-      if (sampleSiteForExperiment(`QUERY_ON_DEMAND`, 10)) {
-        const semverConstraints = {
-          // Because of this, this flag will never show up
-          "gatsby-plugin-sharp": `>=2.10.0`,
-        }
-        if (satisfiesSemvers(semverConstraints)) {
-          return `OPT_IN`
-        } else {
-          // gatsby-plugi-sharp is either not installed or not new enough so
-          // just disable — it won't work anyways.
-          return false
-        }
-      } else {
-        return true
-      }
-    },
-  },
```

Comment:
Do we want to remove it completely or change its behaviour? Removing it will result in "unknown flags" messages when it's used in config, while it would be tad nicer to mention that this is no longer configurable etc (flag graduated into default)

Reply:
I've reverted all changes to flags.ts as we discussed